src/app/components/sidebar/filters/Filters.tsx

```diff
@@ -1,29 +1,65 @@
 import * as Styled from './Filters.styles';
-import { useIntl } from 'react-intl';
 import { Condition } from './condition/Condition';
 import { LabelWithButton } from 'ui';
 import { FiltersProps } from './Filters.types';
+import { Price } from './price/Price';
+import { useLocale } from 'hooks';
+import { FormatNumberOptions } from 'react-intl';
+
+export const Filters = ({ onConditionClick, onPriceChange, activeFilters }: FiltersProps) => {
+  const { formatMessage, formatNumber } = useLocale();
+
+  const currencyConfig: FormatNumberOptions = {
+    style: 'currency',
+    currency: 'PLN',
+    maximumFractionDigits: 0,
+  };
 
-export const Filters = ({ onConditionClick, activeFilters }: FiltersProps) => {
-  const { formatMessage } = useIntl();
   return (
     <>
-      {activeFilters.condition && activeFilters.condition.length > 0 && (
+      {((activeFilters.condition && activeFilters.condition.length > 0) ||
+        activeFilters.minPrice ||
+        activeFilters.maxPrice) && (
         <Styled.FiltersApplied>
-          {activeFilters.condition.map((condition) => (
+          {activeFilters.condition &&
+            activeFilters.condition.length > 0 &&
+            activeFilters.condition.map((condition) => (
+              <LabelWithButton
+                key={condition}
+                onClick={() => {
+                  onConditionClick(condition);
+                }}
+                text={formatMessage({ id: `singleProduct.condition.${condition}` })}
+              />
+            ))}
+          {activeFilters.minPrice && (
+            <LabelWithButton
+              onClick={() => {
+                onPriceChange({ minPrice: undefined });
+              }}
+              prefix={`${formatMessage({ id: `filters.price` }).toLowerCase()} ${formatMessage({
+                id: `filters.minPrice`,
+              })}`}
+              text={formatNumber(activeFilters.minPrice, currencyConfig)}
+            />
+          )}
+          {activeFilters.maxPrice && (
```

Comment:
Tak patrząc na te trzy dodawania komponentu `LabelWithButton`... Właściwie trzy razy robisz prawie to samo, prawda?
Zamiast tego można przed returnem stworzyć arajkę nazwaną np `activeFliterItems`, która będzie miała w sobie obiekty w stylu `{ onClick, prefix, text }`.